CONTRIBUTING.md

```diff
@@ -21,6 +21,7 @@ $GOPATH/src/github.com/wangkuiyi/gotorch/cgotorch/build.sh
 Run the Go examples and unit tests.
 
 ```bash
+cd $GOPATH/src/github.com/wangkuiyi/gotorch
```

Comment:
Actually, we don't need the `cd` command to make `go test` run.

Reply:
Well. On my machine, directly run `go test` failed but succeed after `cd`. If we want `go test` to work outside the directory where go.mod is on, maybe a `-t` argument for `go get` is needed. I would prefer `cd` if we already use `go mod`. The terminal output was attached to the original post.

Reply:
I see. You are right -- GoTorch doesn't only rely on the Go toolchain; it also depends on GNU make and the C++ code.

Thanks for correcting me!